src/main/java/uk/ac/ebi/eva/pipeline/jobs/AnnotationJob.java

```diff
@@ -42,22 +45,24 @@
 @Configuration
 @EnableBatchProcessing
 @Import({AnnotationFlow.class})
-public class AnnotationJob extends CommonJobStepInitialization {
-    public static final String jobName = "annotate-variants";
+public class AnnotationJob {
 
-    @Autowired
-    private JobBuilderFactory jobBuilderFactory;
+    private static final Logger logger = LoggerFactory.getLogger(AnnotationJob.class);
+
+    public static final String NAME_ANNOTATE_VARIANTS_JOB = "annotate-variants-job";
 
     @Autowired
-    private Flow annotationFlowBasic;
+    @Qualifier(AnnotationFlow.NAME_VEP_ANNOTATION_FLOW)
+    private Flow annotation;
+
+    @Bean(NAME_ANNOTATE_VARIANTS_JOB)
+    public Job annotateVariantsJob(JobBuilderFactory jobBuilderFactory) {
+        logger.debug("Building '"+NAME_ANNOTATE_VARIANTS_JOB+"'");
```

Comment:
There should be whitespaces between the strings and the plus signs. Applies to the rest of jobs.